packages/devtools/lib/src/tables.dart

```diff
@@ -209,24 +229,29 @@ class Table<T> extends Object with SetStateMixin {
       }
     }
 
-    data.sort((T a, T b) {
-      if (numeric) {
-        final num one = column.getValue(a);
-        final num two = column.getValue(b);
-        if (one == two) {
-          return 0;
-        }
-        if (_sortDirection == SortOrder.ascending) {
-          return one > two ? 1 : -1;
-        } else {
-          return one > two ? -1 : 1;
-        }
+    _sortData(column, direction);
+  }
+
+  void _sortData(Column column, int direction) =>
+      data.sort((T a, T b) => _compareData(a, b, column, direction));
+
+  int _compareData(T a, T b, Column column, int direction) {
+    if (column.numeric) {
+      final num one = column.getValue(a);
+      final num two = column.getValue(b);
+      if (one == two) {
```

Comment:
use https://api.dartlang.org/stable/2.3.2/dart-core/num/compareTo.html
to simplify this logic.
in the descending case you just need to take
-one.compareTo(two);